v2/stacks/feedback/upstream-services.yml

```diff
@@ -0,0 +1,14 @@
+# Upstream Services
+
+# Contains all the upstream services for the feedback stack
```

Comment:
This isn't technically true. 

We could put a list here of other services you could put in if you want. 
```suggestion
# Contains examples of upstream services for the feedback stack
# Other services which use the feedback footer include:
# - dp-frontend-homepage-controller 
```
...etc.

Reply:
Done